index.js

```diff
@@ -1,17 +1,81 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    //Question to teacher: Why we cannot write the next two code lines in the same line? (this.items.push(item).sort((a,b) => a-b))
+    this.items.push(item);
+    this.items.sort((a,b) => a-b);
 
-  get(pos) {}
+    this.length ++;
+  }
 
-  max() {}
+  get(pos) {
+    if(pos >= this.items.length){
+      throw new Error('OutOfBounds');
+    } 
+    
+    else{
+      return this.items[pos];
+    }
+  }
 
-  min() {}
+  max() {
+    if(this.length === 0){
```

Comment:
Te voy a dar otra forma de hacerlo para que la pienses, aunque esta esta perfecta!!

```suggestion
    if(!this.length){
```

Mirala y me dices si te cuadra (solo te doy una pista, como se evalua el 0??)